src/components/Link/Link.scss

```diff
@@ -30,6 +30,12 @@
   overflow: inherit;
   text-overflow: inherit;
 
+  &.is-disabled {
+    color: $ms-color-neutralTertiary;
+    pointer-events: none;
+    cursor: pointer;
```

Comment:
good catch! Seems with pointer-events: none, you get a default cursor regardless, so it worked fine in testing. 

Updated in the case pointer-events isn't supported.